src/api/XRProjectionLayer.ts

```diff
@@ -272,12 +271,8 @@ export default class XRProjectionLayer extends XRCompositionLayerPolyfill {
 		// initialize layer.isStatic = false
 		this.isStatic = false
 
-		// TODO: XR Compositor also needs to make use of depth values
```

Comment:
Was this commit (to ignore all depth buffer) intentional in this PR? I noticed you had closed a PR that tried to do this same thing.

Reply:
That commit did it the other way. I forgot that Hololens (and maybe others) require depth pixels